### PR TITLE
Included blog post on Lars's appointment to the Advisory Board

### PR DESCRIPTION
**Pull request by propensive**
Review by @heathermiller

### CODE REVIEW

**Comment by heathermiller** [comment]
This looks really nice guys. I'm really glad that Lars has been selected by the Typelevel group to sit on the board. I **love** that you did this democratically, btw. I can't put into words how great that is :)

One small issue though; as per @odersky's email last week, the Typelevel rep must be confirmed at the net AB meeting. So perhaps the wording should reflect this?

File: blog/_posts/2016-10-17-lars-hupel-joins-sc-board.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+work is focussed around the Cats, Shapeless and Spire libraries, but has grown
+to include over a dozen other libraries, tools and compiler plugins, all with
+a shared philosophy.
+
+Typelevel have been instrumental in helping to grow the Scala community, and
+the functional elements within it, through regular hacking sessions, and with a
+number of workshops and unconferences colocated with larger Scala events.
+
+So we felt it was only appropriate — given Typelevel's influence, its track
+record, and the pholosophy of welcomeness it shares with the Scala Center —
```
Comment (heathermiller) [inline blog/_posts/2016-10-17-lars-hupel-joins-sc-board.md]: <3

Comment (dwijnand) [inline blog/_posts/2016-10-17-lars-hupel-joins-sc-board.md]: philosophy

**Comment by propensive** [comment]
Ready to go?

**Comment by larsrh** [comment]
Good from my end!

**Comment by dwijnand** [comment]
There's a typo. s/pholosophy/philosophy/

**Comment by heathermiller** [comment]
Yep, just Dave's typo, but otherwise ready to go! :)

**Comment by SethTisue** [comment]
typo fixed in fd8264b

**Comment by heathermiller** [comment]
Hey Seth, this wasn't quite ready due to the time it has taken to merge. It's now buried under several other announcements. Due to the Scalafix announcement today, I was planning on fixing the dates and announcing this tomorrow.

**Comment by heathermiller** [comment]
We can leave things as-is, since, unfortunately the blog post welcoming Lars is buried. But tomorrow I plan shift this post to the top and to break out all the cowbells I can find to yell and scream and let everyone know...

**Comment by larsrh** [comment]
@heathermiller Please ping me if you do so ... I thought this was legit, so I released the @typelevel post as well and linked to that one. Moving it up will probably change the URL, I guess?

**Comment by heathermiller** [comment]
Oops. Damn. Ok, I'll fix this and since @MasseGuillaume isn't yet finished with some tweaking for the Scalafix release, we can bump that to tomorrow.

**Comment by heathermiller** [comment]
Hang tight, @larsrh. We can sync on this.

**Comment by larsrh** [comment]
@heathermiller No problem, tell me when you're ready.

**Comment by heathermiller** [comment]
Ok, reshuffled and tweeted: https://twitter.com/scala_lang/status/790594241718288384

**Comment by larsrh** [comment]
Thanks, @heathermiller. The updated link will appear in two minutes on our page.

**Comment by SethTisue** [comment]
sorry Heather!

**Comment by heathermiller** [comment]
No worries, it's all good, we figured it out :) We're going to hold off announcing Scalafix until tomorrow. All good :)

**Pull request merged**